mergin/client.py

```diff
@@ -618,3 +626,91 @@ def get_projects_by_names(self, projects):
 
         resp = self.post("/v1/project/by_names", {"projects": projects}, {"Content-Type": "application/json"})
         return json.load(resp)
+
+    def download_file(self, project_path, file_path, output_filename, version=None):
```

Comment:
Hmm after the complete review I am thinking if it won't be better to require local project_dir in this function - so that we can keep the assumption in the "pull" code that MerginProject instance always exists - things would be simpler... what do you think?